### PR TITLE
Improve the handling nullValue of @Compare #372

### PR DESCRIPTION
**Pull request by btyoshikawaa**
For response of this issue comment, I sent this pull request.
https://github.com/terasolunaorg/terasoluna-gfw/issues/372#issuecomment-156920292 About  `null`  value
https://github.com/terasolunaorg/terasoluna-gfw/issues/372#issuecomment-160121086

Please review #372.

### CODE REVIEW

File: terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraints/Compare.java [hunk 1]
```diff
@@ -79,6 +79,12 @@
     Operator operator();
 
     /**
+     * @return {@code true} it's valid if left and right are both null, but it's invalid if either is null. otherwise it's valid
```
Comment (ikeyat) [inline terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraints/Compare.java]: - [ ] @return configure how to treat {@code null} value in given parameters. If true, it is regarded as valid only when both left and right are {@code null}. It is regarded as invalid when either is {@code null}. If false, both cases are regarded as valid.

File: terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraintvalidators/CompareValidator.java [hunk 2]
```diff
@@ -52,6 +52,12 @@
     private Operator operator;
 
     /**
+     * {@code true} it's valid if left and right are both null, but it's invalid if either is null. otherwise it's valid even if
+     * either is null.
```
Comment (ikeyat) [inline terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraintvalidators/CompareValidator.java]: - [ ] configure how to treat {@code null} value in given parameters. If true, it is regarded as valid only when both left and right are {@code null}. It is regarded as invalid when either is {@code null}. If false, both cases are regarded as valid.

**Comment by ikeyat** [comment]
See this comment
https://github.com/terasolunaorg/terasoluna-gfw/issues/372#issuecomment-159236137

**Comment by btyoshikawaa** [comment]
@ikeyat 
Thanks for your comments.
I've reflected in the javadoc. Please show 668bc68f0b7624153796f2027c242c2ce017a3e8.

File: terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraints/Compare.java (hunk 1 above)
Comment (btyoshikawaa) [inline terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraints/Compare.java]: @ikeyat 
Thanks for your comments.
I've reflected in the javadoc. Please show 668bc68f0b7624153796f2027c242c2ce017a3e8.

File: terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraintvalidators/CompareValidator.java (hunk 2 above)
Comment (btyoshikawaa) [inline terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraintvalidators/CompareValidator.java]: @ikeyat 
Thanks for your comments.
I've reflected in the javadoc. Please show 668bc68f0b7624153796f2027c242c2ce017a3e8.

**Pull request merged**